packages/sdk/src/protocols/filter.ts

```diff
@@ -381,7 +418,7 @@ class FilterSDK extends BaseProtocolSDK implements IFilterSDK {
     return toAsyncIterator(this, decoders);
   }
 
-  private getPubsubTopics<T extends IDecodedMessage>(
+  private getUniquePubsubTopics<T extends IDecodedMessage>(
```

Comment:
why `unique`? it just returns `pubsubTopics` without duplicates

Reply:
yes, so it only returns _unique_ pubsub topics, right? ^.^